client/my-sites/plugins/plugin-list-header/index.jsx

```diff
@@ -38,31 +39,22 @@ function checkPropsChange( nextProps, propArr ) {
 	return false;
```

Comment:
Would it make sense to move `checkPropsChange` as a method of the `PluginsListHeader` class? I don't think there's a good reason to keep it as a separate function.

Also, this method of handling props changes feels a little old, but I'll comment that under the `shouldComponentUpdate` definition.